api/tests/Feature/PoolCandidateUpdateTest.php

```diff
@@ -1051,20 +1053,20 @@ public static function manualStatusProvider()
                 PoolCandidateStatus::QUALIFIED_AVAILABLE->name,
                 'final_decision_at',
             ],
+            'qualified unavailable sets final decision' => [
+                PoolCandidateStatus::QUALIFIED_UNAVAILABLE->name,
+                'final_decision_at',
+            ],
+            'qualified withdrew sets final decision' => [
+                PoolCandidateStatus::QUALIFIED_WITHDREW->name,
+                'final_decision_at',
+            ],
 
             // Removed
```

Comment:
I think you're missing SCREENED_OUT_NOT_INTERESTED here.

Reply:
Kind of out of scope.  Eric will complete this if there is time.